docs/components/animation.md

```diff
@@ -112,6 +112,38 @@ Accessed as `el.components.animation.<MEMBER>`.
 | animation | anime.js object.           |
 | config    | Config passed to anime.js. |
 
+### Controlling Animations using setAttribute
+
+Like any A-Frame component, the animation component can be configured from JavaScript by calling setAttribute() on an element.
+
+The second parameter can either be a string, or an object, detailing the properties to set.
```

Comment:
We should probably link to the relevant docs to avoid duplication https://aframe.io/docs/1.2.0/introduction/javascript-events-dom-apis.html#updating-a-component-with-setattribute

Reply:
ok - fixed.